src/test/java/calculator/PositiveNumberTest.java

```diff
@@ -0,0 +1,27 @@
+package calculator;
+
+import calculator.exception.NegativeNumberException;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class PositiveNumberTest {
+
+    @DisplayName(value = "양수로 PositiveNumber 를 생성할수 있다.")
+    @ParameterizedTest
+    @CsvSource(value = {"1,2"}, delimiter = ',')
```

Comment:
`@ValueSource`를 사용하면 어떨까요? 아래의 글을 참고하세요.
https://www.baeldung.com/parameterized-tests-junit-5

```suggestion
    @ValueSource(ints = {1, 2})
```